package.json

```diff
@@ -0,0 +1,81 @@
+{
+    "name": "hmis-dictionary",
+    "description": "DHIS2 HMIS Dictionary app",
+    "version": "1.3.2",
+    "license": "GPL-3.0",
+    "author": "EyeSeeTea team",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/EyeSeeTea/HMIS-Dictionary.git"
+    },
+    "homepage": ".",
+    "dependencies": {
+        "@bower_components/angucomplete-alt": "ghiden/angucomplete-alt#2.5.0",
+        "@bower_components/angular": "angular/bower-angular#1.5.9",
+        "@bower_components/angular-resource": "angular/bower-angular-resource#1.5.9",
+        "@bower_components/angular-sanitize": "angular/bower-angular-sanitize#1.5.9",
+        "@bower_components/angular-table": "file:node_modules/@bower_components/angular-table",
+        "@bower_components/angular-translate": "PascalPrecht/bower-angular-translate#~2.11.0",
+        "@bower_components/angular-translate-loader-static-files": "PascalPrecht/bower-angular-translate-loader-static-files#2.11.0",
+        "@bower_components/angular-ui-bootstrap": "angular-ui/bootstrap-bower#2.3.0",
+        "@bower_components/angular-ui-router": "angular-ui/angular-ui-router-bower#0.3.2",
+        "@bower_components/angular-ui-tinymce": "angular-ui/ui-tinymce#0.0.17",
+        "@bower_components/bootstrap": "twbs/bootstrap#3.3.7",
+        "@bower_components/bootstrap-vertical-tabs": "dbtek/bootstrap-vertical-tabs#1.2.1",
+        "@bower_components/d3": "mbostock-bower/d3-bower#4.4.0",
+        "@bower_components/jquery": "jquery/jquery-dist#2.2.4",
+        "@bower_components/lodash": "lodash/lodash#4.17.11",
+        "@bower_components/tinymce": "tinymce/tinymce-dist#4.5",
+        "d2-manifest": "1.0.0"
+    },
+    "devDependencies": {
+        "http-server": "^14.1.1",
+        "prettier": "2.6.1"
+    },
+    "scripts": {
+        "postinstall": "sh build-bower-deps.sh",
+        "build-folder": "rm -rf build/ && d2-manifest package.json manifest.webapp && mkdir build && yarn run manifest && cp -r bower_components app assets languages doc assets/img/Logo_48.png index.html build",
+        "build": "yarn build-folder && rm -f $npm_package_name.zip && cd build && zip --quiet -r ../$npm_package_name.zip *",
+        "start": "export $(xargs < .env.local) && http-server . -a $HOST -p $PORT -c-1 -P $REACT_APP_DHIS2_BASE_URL --proxy-options.auth $REACT_APP_DHIS2_AUTH",
```

Comment:
`source .env.local &&` should also work.

Reply:
For `source` to work the default shell used by Yarn have to support it. This can be checked with `yarn config get script-shell`. If undefined looks like it fallbacks to `sh`.
Ubuntu uses `dash` for it `sh` command, and it doesn't support `source` .
So, a "universal" alternative to `source .env.local &&` is `. ./.env.local &&` (the `./` is required by `dash`)